docs/release-notes/NuGet-5.7.md

```diff
@@ -0,0 +1,90 @@
+---
+title: NuGet 5.7 Release Notes
+description: Release notes for NuGet 5.7 including new features, bug fixes, and DCRs.
+author: chgill-msft
+ms.author: chgill
+ms.date: 8/14/2020
+ms.topic: conceptual
+---
+
+# NuGet 5.7 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**5.7.0**](https://nuget.org/downloads) | [Visual Studio 2019 version 16.7](https://visualstudio.microsoft.com/downloads/) | [3.1.401](https://dotnet.microsoft.com/download/dotnet-core/3.1)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2019 with .NET Core workload
```

Comment:
This seems redundant. Although it's included in VS 2019, we say that above with `Visual Studio 2019 version 16.7`.

We can change the column to **Available in .NET Core SDK(s)** to make it clear it's Core.

Is there a difference if a user has VS 2019 16.7 & does not install the .NET Core workload? What version do they get?

Reply:
They get the same NuGet version, but not the dotnet SDK.